mrblib/haconiwa/base.rb

```diff
@@ -568,6 +568,9 @@ class Capabilities
       cap_dac_override
       cap_fowner
       cap_fsetid
+      cap_net_bind_service
+      cap_net_broadcast
+      cap_net_admin
```

Comment:
`cap_net_admin` looks too strong... It allows containers' root to modify IF, configure routings and so on.
I want to know what kind of background( or use case) for this cap.
Other 2 may be good.

Reply:
It was too rough. As you say, adding only net_bind_service seems good.